includes/collection/class-outbox.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Outbox collection file.
+ *
+ * @package Activitypub
+ */
+
+namespace Activitypub\Collection;
+
+/**
+ * ActivityPub Outbox Collection
+ */
+class Outbox {
+	const POST_TYPE = 'ap_outbox';
+
+	/**
+	 * Add an Item to the outbox.
+	 *
+	 * @param \Activitypub\Activity\Base_Object $activity_object    The Activity-Object  to add as JSON.
+	 * @param string                            $activity_type      The activity type.
+	 * @param int                               $user_id            The user ID.
```

Comment:
The real or imaginary user ID of the actor that published the activity that will be added to the outbox.
```suggestion
	 * @param int                               $user_id            The real or imaginary user ID of the actor that published the activity that will be added to the outbox.
```